integration_tests/fixtures/forseti/.gitignore

```diff
@@ -0,0 +1,3 @@
+terraform.tfvars
```

Comment:
Why have a local `.gitignore` file here?  Is there any consideration why this should not be in the top-level `.gitignore` file?

Reply:
It can certainly be consolidated, and it has been consolidated.